content/22.vaccines.md

```diff
@@ -247,6 +255,11 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Protein Subunit Vaccines
 
 <!--Do we need to add this to the figure? Or does it fall in one of the other categories?-->
+Compared to the _inactivated whole virus vaccine_, these protein subunit vaccines isolate a single protein of the virus and use it to stimulate the immune system.
+These proteins, also referred to as antigens, are usually those located on the surface of the viral particle and therefore key targets of the immune system.
+These proteins are typically grown in yeast and then harvested.
+Because the vaccine contains only the purified antigens, the vaccine can only stimulate an antibody response [@doi:10.1007/978-1-4615-1823-5].
+The main advantage of this method is that they are considered very safe  since the anteigen alone cannot cause an infection, however the immune response is weaker and usually an adjuvant is needed to boost the response [@doi:10.1517/14712598.2011.573624].
```

Comment:
```suggestion
The main advantage of this method is that they are considered very safe because the antigen alone cannot cause an infection; however, the immune response is weaker and an adjuvant is usually needed to boost the response [@doi:10.1517/14712598.2011.573624].
```